<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailedWeatherDialog</class>
 <widget class="QDialog" name="DetailedWeatherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_title">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Szczegółowa pogoda - London</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_hour">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(195, 108, 109);
}</string>
       </property>
       <property name="text">
        <string>17:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_info">
     <item row="0" column="0">
      <widget class="QLabel" name="label_tempTitle">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Temperatura: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_feelTempTitle">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Temperatura odczuwalna:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_windSpeedTitle">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Prędkość wiatru (km/h):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_humidityTitle">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wilgotność powietrza:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_rainChanceTitle">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Szansa na deszcz:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_snowChanceTitle">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Szansa na śnieg:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_temp">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(38, 220, 74);
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_feelTemp">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(38, 220, 74);
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_windSpeed">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(38, 220, 74);
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_humidity">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(38, 220, 74);
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_rainChance">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(38, 220, 74);
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_snowChance">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(38, 220, 74);
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
